pkg/planner/core/casetest/index/index_test.go

```diff
@@ -148,3 +148,15 @@ func TestRowFunctionMatchTheIndexRangeScan(t *testing.T) {
 		tk.MustQuery(tt).Sort().Check(testkit.Rows(output[i].Result...))
```

Comment:
Please run the ```make bazel_prepare``` and update the changement.

Reply:
@hawkingrei thanks! unfortunately @ari-e is out for a bit and I don't have permissions to update this PR, so I've addressed the comment separately in #54290